apps/studio/components/interfaces/ExplainVisualizer/ExplainVisualizer.NodeRenderer.tsx

```diff
@@ -0,0 +1,152 @@
+import { ArrowRight, ArrowUp, Clock } from 'lucide-react'
+import { cn } from 'ui'
+import { CostIndicator } from './ExplainVisualizer.CostIndicator'
+import { RowCountIndicator } from './ExplainVisualizer.RowCountIndicator'
+import {
+  getOperationColor,
+  getOperationDescription,
+  getOperationIcon,
+} from './ExplainVisualizer.utils'
+
+import type { ExplainNode } from './ExplainVisualizer.types'
+
+export interface ExplainNodeRendererProps {
+  node: ExplainNode
+  isLast: boolean
+  depth: number
+  maxCost: number
+  parentHasMore: boolean[]
```

Comment:
are `parentMore` and `isLast` actually used? I see them passed through the recursion but couldn't find anywhere in the component that actually uses them